includes/widgets/class-amp-widgets.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Register widgets, and add actions.
+ *
+ * @package AMP
+ */
+
+/**
+ * Register the widgets.
+ */
+class AMP_Widgets {
+
+	/**
+	 * Add the actions.
+	 *
+	 * @return void.
+	 */
+	public function init() {
+		add_action( 'widgets_init', array( $this, 'register_widgets' ) );
+		add_action( 'show_recent_comments_widget_style', '__return_false' );
+		add_action( 'wp_footer', array( $this, 'dequeue_scripts' ) );
```

Comment:
Per below, this shouldn't be necessary.

Reply:
Thanks, this [commit](https://github.com/Automattic/amp-wp/pull/870/commits/be58d0243f1ef4e3533e853ff6bb436231104d4f#diff-4a5a57759183e45d3afe37b6f63e406cL21) deletes this `add_action()` call.